docker-compose/start-docker-compose.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+export DOCKER_HOST_IP=$(ip route | grep docker0 | awk '{print $9}')
+docker-compose -f ./travel-agency-services.yml rm
```

Comment:
@cristianonicolai just a minor comment here. For me, it's needed remove the old containers to make it work, I'm guessing it could be a good idea add something like ' yes y | docker-compose -f ./travel-agency-services.yml rm'